src/components/Accordion/Accordion.tsx

```diff
@@ -37,7 +37,11 @@ export default function Accordion(
             id={id}
             className={bem("accordion", modifiers, blockName, [className])}
         >
-            <input id={`accordion-${inputId}`} type="checkbox" />
+            <input
+                id={`accordion-${inputId}`}
```

Comment:
This might not be a good change for this PR, which is fine—but I dislike prepending the IDs with a string, I think it's a little unclear for consumers to have the ID they pass in a prop not be the ID-hook they'll need in other places. Do you have an opinion on this @hokei?

Reply:
This pattern is used in other components. I did get confused at first when using some components because the output ID was not what I passed in. It makes sense for the association done with the label. You're right that if this is redone, it should be a new ticket/PR.

Reply:
I can file tickets for this work per component if @hokei agrees it's work worth doing.

Reply:
I want to first know if this is a thing people will need.  

For most users, the fact that Accordion is also a checkbox is not a thing they need to care about, and if it's possible to have this "just work" without needing to know the internals then I say we should go for it.  But if someone finds that they absolutely need to access this we should change it then.

Reply:
Oh yikes, I'm seeing that I've gotten turned around—this is basically a "local" id. 

What I originally was questioning was whether external, consumer ids should ever be prepended in any way. I'd still argue not. If/when we have any of those left, let's remove any prepended string(s), but now looking at this properly this isn't one of those cases. Ignore me!